

class ContextException(Exception):
    pass


class TaskNotFoundException(ContextException):
    pass

